### ISSUE TITLE
[ADE Support] Critical Defect 1 WCAG 1.3.1 Information and Relationships 

### ISSUE THREAD

**Issue by lakper** [department-of-veterans-affairs/va.gov-team#106459]
> [!NOTE]
> **QUICK SUMMARY:** _Ticket created to track feedback on 508 audit findings for ticket #106395 ._

**Defect 1 | Critical | Created:**
_WCAG 1.3.1 Information and Relationships | Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text._

On the COLUMBIA UNIVERSITY IN THE CITY OF NEW YORK: GI Bill® Comparison Tool | Veterans Affairs screen, when navigating with tab, the current page link is visually distinguishable from the other links but this cannot be understood by JAWS—“Institution details Link” is read. (New: 03/31/2025)

On the COLUMBIA UNIVERSITY IN THE CITY OF NEW YORK: GI Bill® Comparison Tool | Veterans Affairs screen, the structure of information cannot be understood by assistive technology. The text, starting with 4 year program, is read as unstructured text—one might use a list. (New: 03/31/2025)

On the COLUMBIA UNIVERSITY IN THE CITY OF NEW YORK: GI Bill® Comparison Tool | Veterans Affairs screen, the table structure cannot be understood by assistive technology. For example, when reading the first table, the row header is not read for the second row and “Empty header All student complaints Column 1” is read when reading column one—“All student complaints Column 1” should be read. (New: 03/31/2025)

Project Name: Yellow Ribbon Program information in Comparison Tool VASI #: 2103  
Audit Number: 43731 Audit Date: 3/31/2025
Audit Scope: General
Product Type: Web Applications
Project Status: Open
Project Site: https://www.va.gov/education/gi-bill-comparison-tool/institution/31937132
Project Version:
Release Recommendation: No
Release Recommendation Comments: The defects are listed in the following report.
Link to 508 Compliance Application: [508 Compliance Application](https://apps.gov.powerapps.us/play/e/default-e95f1b23-abaf-45ee-821d-b7ab251ab3bf/a/5f7da63e-384d-4ce4-8cc7-f3be6c60c363?tenantId=e95f1b23-abaf-45ee-821d-b7ab251ab3bf&sourcetime=2023-08-14%2017%3A50%3A55Z&RecordID=2673)

### Ticket Updates
- [x] _Connect to an `Epic/Parent` or `Intake` (what body of work is this a part of?) If this is the `Epic/Parent`, connect related tickets as sub-issues._
- [ ] _If this is the `Epic/Parent` ticket, add label for `ADE 508 RECHECK`_
- [x] _If this ticket is a `Child` ticket aka `sub-issue` please add the appropriate labels for `ADE 508 FLAG CRITICAL`;`ADE 508 FLAG HIGH`; `ADE 508 FLAG MEDIUM` or `ADE 508 FLAG LOW`_
- [x] _Label with `Originator/Team` (product team or stakeholder requesting support; ADE if internal)_
- [x] _Label date in the `Open Date` field_
- [ ] _Label with `Estimate` (level of effort expected for this ticket)_
- [ ] _Add `Assignee(s)` name(s) to ticket_ (include Lakwi)
- [ ] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [ ] _Select a `Priority Level`_
- [x] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`
- [ ] _Make sure the `ADE 508 RECHECK` label is on this ticket upon closing and that a follow-up date has been established and calendared._

**Comment by coforma-jamie** [comment]
[Findings in parent ticket comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/106395#issuecomment-2773380490)
